tracpro/polls/models.py

```diff
@@ -635,7 +635,7 @@ def numeric_group_by_date(self):
                 # Append the sum total value for each date
                 if total:
                     answer_sums.append(total)
-                    answer_averages.append(float(total/total_answers))
+                    answer_averages.append(round(float(total/total_answers), 2))
```

Comment:
Change looks good. On an unrelated note the `float(...)` isn't doing anything since `total` is defined as a float above in line 630.